images/fedora/f28/missing-docs

```diff
@@ -0,0 +1,22 @@
+acl
+bash
+chkconfig
+curl
+dnf
+gawk
+glusterfs
```

Comment:
Typo? :)

Did we really have `glusterfs` in the F28 toolbox at some point in time? At least right now, I don't see a `glusterfs` package anymore.

Reply:
Removed.